yoyodyne/datasets.py

```diff
@@ -465,9 +453,6 @@ def _iter_samples(
                 features = self._get_cell(
                     row, self.features_col, self.features_sep
                 )
-                # Use unique encoding for features.
-                # This disambiguates from overlap with source.
-                features = [f"[{feature}]" for feature in features]
```

Comment:
Are we no longer encoding features with unique formatting?

Reply:
This is a merge error of course, fixed.